Source/Logger.h

```diff
@@ -60,7 +60,7 @@ inline void Log(const spdlog::source_loc &srcloc, Args &&...args)
             return spdlog::level::critical;
         }
         else {
-            static_assert(false);
+            assert(false);
```

Comment:
This PR should not contains this unrelated change, I'll fix it in later commits.